src/ButteredBread.sol

```diff
@@ -141,7 +141,7 @@ contract ButteredBread is ERC20VotesUpgradeable, OwnableUpgradeable, IButteredBr
         if (_accountToLPData[_account][ZERO_ADDRESS].balance != 1) {
             /// @dev truthy value to check if account has ever made a deposit
             _accountToLPData[_account][ZERO_ADDRESS].balance = 1;
-            depositors.push(_account);
+            _depositors.push(_account);
```

Comment:
missing this logic on the withdraw 

this means that we need to implement logic for checking if the withdraw is a full or partial withdraw request

Reply:
Also this doesn't account for the different kinds of depositors, we need a mapping of the lp address to the `depositors` list

Reply:
yeah, that's why it was just added once ever, despite LP type, so there would not be duplicates. anyway, i merged so this is irrelevant now